ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/HUD/flag-48x48.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "ic_settings_24_outlined.pdf",
+      "filename" : "ic_fluent_flag_48_regular.pdf",
```

Comment:
Why did so many of these sizes change? Is there a standard size we should be using for these assets?

Reply:
github is deceiving. so tabbarview had its own settings.pdf so i removed it. HUD technically i think the size is supposed to be 40x40 however this flag is used for "custom" case and there isn't any 40x40 icon in fluent system icon so i just picked 48x48 in this case.